clients/src/main/resources/common/message/ElectLeadersRequest.json

```diff
@@ -28,7 +28,7 @@
     { "name": "TopicPartitions", "type": "[]TopicPartitions", "versions": "0+", "nullableVersions": "0+",
       "about": "The topic partitions to elect leaders.",
       "fields": [
-        { "name": "Topic", "type": "string", "versions": "0+", "entityType": "topicName",
+        { "name": "Topic", "type": "string", "versions": "0+", "entityType": "topicName", "mapKey": true,
           "about": "The name of a topic." },
         { "name": "PartitionId", "type": "[]int32", "versions": "0+",
```

Comment:
nit: While we are here, should we rename this field to `Partitions`? I was confused by this while reading the code. When I read `PartitionId`, I don't really expect an array.

Reply:
make sense